test/integration/test_build.py

```diff
@@ -25,3 +26,16 @@ def test_build_streams_output(cli, container_runtime, build_dir_and_ee_yml, ee_t
     result = cli(f"ansible-builder build -c {tmpdir} -f {eeyml} -t {ee_tag} --container-runtime {container_runtime}")
     assert f"{container_runtime} build -f {tmpdir}" in result.stdout
     assert f"The build context can be found at: {tmpdir}" in result.stdout
+
+
+def test_user_system_requirement(cli, container_runtime, ee_tag, tmpdir, data_dir):
```

Comment:
does this happily just run twice and the test and `ee_tag` get same value from `container_runtime` fixture?

Reply:
It did seem so, it was comforting that it ran the same number of tests, it didn't crazily multiple them 2x